README.md

```diff
@@ -101,6 +101,200 @@ any zone.
 
 You can define an arbitrary number of drag items.
 
+Analytics Events
+----------------
+
+The following analytics events are provided by this block.
+
+## `xblock.drag-and-drop-v2.loaded`
+
+Fired when the Drag and Drop XBlock is finished loading.
+
+### Example
+
+```
+{
+    "username": "staff",
+    "event_type": "xblock.drag-and-drop-v2.loaded",
```

Comment:
@Kelketek I think you'll need to adjust these `"event_type"` entries to use underscores instead of dashes.